tools/debian/control

```diff
@@ -0,0 +1,116 @@
+Source: cockpit
+Section: admin
+Priority: optional
+Maintainer: Michael Biebl <biebl@debian.org>
```

Comment:
I think Michael said he didn't want to take this on right now.

@mbiebl Can we leave you in here for now?